### PR TITLE
Mobile - Endpoint details page #316

### PR DESCRIPTION
**Pull request by Thacryba**

### CODE REVIEW

File: src/pages/endpoint/Endpoint.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      left: 6px;
+      margin-top: -10px;
+    }
+
+    svg {
+      margin-left: 16px;
+    }
+  }
+
+  .title-tile {
```
Comment (comountainclimber) [inline src/pages/endpoint/Endpoint.scss:49]: Can we make sure that these labels are centered => 
Screen Shot 2021-05-04 at 8 41 05 AM

File: src/pages/endpoint/Endpoint.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    .replace(/\+/g, 'http://')
+    .replace(/&/g, 'https://')
+    .replace(/_/g, '.')
+    .replace(/-/g, ':')
+
+  const endpointSelected = SerializeNode(nodes).filter(nodes => {
+    return nodes.url === endpointUrl
+  })[0]
+
+  const LOCATIONS_FLAGS = [
```
Comment (comountainclimber) [inline src/pages/endpoint/Endpoint.tsx:38]: Maybe we should import the constant from `Monitor.tsx` for some code reuse

File: src/pages/monitor/Monitor.tsx [hunk 3]
```diff
@@ -107,11 +109,19 @@ const Endpoint: React.FC<Endpoint> = ({ url, locationEndPoint, disable }) => {
...
 }
 
-const IsItUp: React.FC<IsItUp> = ({ statusIsItUp }): JSX.Element => {
+export const IsItUp: React.FC<IsItUp> = ({
+  statusIsItUp,
+  fromMonitor,
+  url,
+}): JSX.Element => {
+  const width = useWindowWidth()
+  const canNavigate = width < 1200 && fromMonitor
```
Comment (comountainclimber) [inline src/pages/monitor/Monitor.tsx:124]: Should we make the endpoint page available on all devices?

**Review by comountainclimber — CHANGES_REQUESTED** [review]
@Thacryba have some minor feedback but overall looks really good! Nice job

File: src/pages/endpoint/Endpoint.scss (hunk 1 above)
Comment (Thacryba) [reply src/pages/endpoint/Endpoint.scss:49]: In the design, the titles were aligned to the top and left, and the information to the right. Then should we make them centered on wider screens and like this on mobile screens?

File: src/pages/monitor/Monitor.tsx (hunk 3 above)
Comment (Thacryba) [reply src/pages/monitor/Monitor.tsx:124]: This page is supposed to complete the information that can't appear on the mobile or tablet version, we can change this too. But should the design be the same on web?

Comment (comountainclimber) [reply src/pages/monitor/Monitor.tsx:124]: lets keep it to mobile and tablet for now 👍

**Pull request merged**